gpytorch/lazy/lazy_tensor.py

```diff
@@ -18,7 +18,6 @@
 from ..utils.cholesky import psd_safe_cholesky
 from ..utils.deprecation import _deprecate_renamed_methods
 from ..utils.getitem import _compute_getitem_size, _convert_indices_to_tensors, _noop_index
-from ..utils.gradients import _ensure_symmetric_grad
```

Comment:
Can probably also rm the file / remove from __init__ if applicable

Reply:
ah yeah I thought this was used somewhere else, but apparently not